pkg/pillar/cmd/zedagent/localinfo.go

```diff
@@ -95,10 +97,10 @@ func localAppInfoPOSTTask(ctx *getconfigContext) {
 
 	for {
 		select {
-		case <-ctx.localAppInfoPOSTTicker.C:
+		case <-ctx.sideController.localAppInfoPOSTTicker.C:
 			start := time.Now()
 			appCmds := postLocalAppInfo(ctx)
-			processReceivedAppCommands(ctx, appCmds)
+			ProcessReceivedAppCommands(ctx, appCmds)
```

Comment:
Is this for LPS only, or is this code used for both LPS and LOC?
Or in other words, where is the call to ProcessReceivedAppCommands made for LOC that I should review?

Reply:
https://github.com/lf-edge/eve/pull/3440/commits/f094ee37db0970cfbc5e7626e1b049798a7a56c4
last function `inhaleCompoundDeviceConfig()` calls both `processing` calls.

Reply:
Then I think it is OK.
Just please check that app and dev commands still work for LPS. You can simply run these eden tests:
```
# in eden repo
 make clean && make build-tests
 ./eden config add default
 ./eden config set default --key eve.tag --value <eve-tag-for-this-pr>
 ./eden config set default --key eve.log-level --value debug
 ./eden setup
 ./dist/bin/eden+ports.sh 2223:2223 2224:2224 5912:5902 5911:5901 8027:8027 8028:8028 8029:8029 8030:8030 8031:8031
 ./eden start
 ./eden eve onboard
 ./eden test tests/eclient -e app_local_info -v debug
 ./eden test tests/eclient -e dev_local_info -v debug
```